app/controllers/api/v1/access_tokens_controller.rb

```diff
@@ -35,6 +35,6 @@ def token_params
   end
 
   def token_deactivate_params
-    params.require(:id)
+    params.require(:token)
```

Comment:
The param is being used to find `AccessToken` by id. This is not the same as a form id. I know all the specs are passing but have you tried this locally to see if it works?

Reply:
I see, I was confused by the fact that this is the only place we refer to the `AccessToken` ID in our code (it's not in the model, or the schema file, but I realise now it's just implied). Thanks for pointing this out, I'll redo this PR.

Reply:
I've changed the rename to be to `token_id`, I've also tweaked the API a bit to be more conventional. I've tested all this in dev.